.github/workflows/docker.yaml

```diff
@@ -73,5 +75,12 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: match-tag
+        with:
+          text: ${{ github.ref_name }}
+          regex: '^(master|main|v[0-9]+\.\d+\.\d+)$'
+
       - name: Notify that new build has been uploaded
+        if: ${{ steps.match-tag.outputs.match != '' }}
         run: curl -X POST https://holy-bread-207a.livepeer.workers.dev
```

Comment:
i wasn't sure how the worker notifications work, and assumed that the current docker tag might get deployed, which is not wanted in all cases.

> just have it be a no-op most of the time since the updated images don't matter to the currently deployed infra.

that makes it easier. i can remove the `match-tag` entirely.